Geographic and Social Predictors of Readmission.Rmd

```diff
@@ -29,8 +32,45 @@ We used bi-directional stepwise selection with the Bayes Information Criterion (
 
 ```{r regression model, echo=FALSE}
 #' echo=FALSE will prevent the printing of the R code that generated the plot
+
 #' [model summary goes here]'
 
+#' Fitting data to appropriate statistic linear model with your predictors
+fit <- lm(_______ ~ _______, data = ________);
+summary(fit);
+#' Provide a clean version of the summary - both produce the same results
+summary(fit) %>% tidy;
+fit %>% tidy;
+#' Show a quick look at all of the summary statistics
+glance(fit);
+
+#' Create a 'null' model that does not specify predictors
+fit0 <- lm(________ ~ 1, data = ________);
+summary(fit0);
+summary(fit0) %>% tidy;
+fit0 %>% tidy;
+glance(fit0);
+
+#' When using lm repeatedly you can use '.~' to keep the original outcome and '~.' to keep the original predictors
+#'
+#' Create an all encompassing model for your predictors
+#' Include some interaction terms with interaction model: 'OUTCOME ~ PRED1*PRED2*PRED3 ...'
+#' For all 2 way interactions : '~ (A+B+C)^2', for all 3 way interactions: '~ (A+B+C)^3'
+#' Additive model for note: 'OUTCOME ~ PRED1 + PRED2 + PRED3 + ...'
+fit1 <- lm(________ ~ ACS_PER_CAPITA_INCOME + ACS_PCT_POSTHS_ED + ACS_MEDIAN_HOME_VALUE + ACS_PCT_MEDICAID_ANY + ACS_PCT_DISABLE + sdi_score, data = ________);
+
```

Comment:
Please comment out any non-working code.

Use `#'` if you want it to show up in the rendered document, `#` otherwise.